src/Pagination/ConnectionField.php

```diff
@@ -85,6 +85,10 @@ public function edgeResolver(Paginator $paginator, array $args, GraphQLContext $
                         $data['node'] = $item;
                         break;
 
+                    case 'pivot':
+                        $data['pivot'] = $item->pivot->getAttributes();
```

Comment:
```suggestion
                        $data['pivot'] = $item->pivot;
```

Allows lazy getters and other particular fields on the pivot model to work.